### PR TITLE
Add an example of passing arrays

### PR DESCRIPTION
**Pull request by adys**

### CODE REVIEW

File: examples/basic/main.tf [hunk 1]
```diff
@@ -46,4 +46,12 @@ module "extenral_dns" {
   cluster_name                     = module.eks_cluster.eks_cluster_id
   cluster_identity_oidc_issuer     = module.eks_cluster.eks_cluster_identity_oidc_issuer
   cluster_identity_oidc_issuer_arn = module.eks_cluster.eks_cluster_identity_oidc_issuer_arn
+
+  settings = {
```
Comment (dojci) [inline examples/basic/main.tf:50]: please add more examples of different nested structure types like:
```
sources:
  # - crd
  - service
  - ingress
  # - contour-httpproxy

```
```
coredns:
  etcdTLS:
    enabled: false
```

**Review by aigamdocs — COMMENTED** [review]
Great, thank you!

File: examples/basic/main.tf (hunk 1 above)
Comment (adys) [reply examples/basic/main.tf:50]: Could you check the other examples I just added?

Comment (dojci) [reply examples/basic/main.tf:50]: good enough

**Review by dojci — APPROVED** [review]

**Comment by adys** [comment]
> Great, thank you!

yw. I'm closing the issue you have opened for this #13 .

**Pull request merged**